### PR TITLE
GH-85: Add section on semantic versioning

### LINKED ISSUE #85: Add section on semantic versioning

### PR DESCRIPTION
**Pull request by Fokko**
Closes #85

### CODE REVIEW

File: content/en/docs/Contribution Guidelines/contributing.md [hunk 1]
```diff
@@ -31,6 +31,30 @@ A pull request can be merged through the GitHub UI. By default, only [squash and
...
+ * @return the corresponding logger
+ * @deprecated will be removed in 2.0.0; use org.slf4j.LoggerFactory instead.
+ */
+@Deprecated
+public static Log getLog(Class<?> c) {
+    return new Log(c);
+}
+```
+
+Checking for any voilations can be done by running `mvn verify`.
```
Comment (wgtmac) [inline content/en/docs/Contribution Guidelines/contributing.md]: `any voilations` looks unclear to me. Running the mvn verify command does not verify missing depreceted annotation or javadoc comment.

BTW, do we need to provide a verbose command like `mvn verify -Dmaven.test.skip=true japicmp:cmp`

Comment (Fokko) [reply content/en/docs/Contribution Guidelines/contributing.md]: Hey Gang, good catch, I agree that it is a bit strong. I've changed it into: 

```
Checking for API violations can be done by running `mvn verify -Dmaven.test.skip=true japicmp:cmp`.
```

**Review by wgtmac — APPROVED** [review]
+1

Thanks @Fokko for adding this!

**Comment by Fokko** [comment]
Thanks for the review @wgtmac

**Pull request merged**